tests/visco_plastic_peierls_mei_viscosity_approx.prm

```diff
@@ -0,0 +1,130 @@
+# A test for the peierls creep viscositiy approximation  
```

Comment:
As Rene left a couple of additional things, I also noticed this:
viscositiy -> viscosity